typescript/sdk/src/contracts/contracts.ts

```diff
@@ -257,3 +260,29 @@ export function appFromAddressesMapHelper<F extends HyperlaneFactories>(
     multiProvider,
   };
 }
+
+export function transferOwnershipTransactions(
+  chainId: number,
+  contract: Address,
+  actual: OwnableConfig,
+  expected: OwnableConfig,
+  label?: string,
+): AnnotatedEV5Transaction[] {
+  if (eqAddress(actual.owner, expected.owner)) {
+    return [];
+  }
+
+  return [
+    {
+      chainId,
+      annotation: `Transferring ownership of ${label ?? contract} from ${
+        actual.owner
+      } to ${expected.owner}`,
+      to: contract,
+      data: Ownable__factory.createInterface().encodeFunctionData(
+        'transferOwnership',
+        [expected.owner],
+      ),
+    },
+  ];
```

Comment:
Me reusing this exact code once this pr is merged

![images (2)](https://github.com/user-attachments/assets/bfb7f4c4-13d8-45be-b87d-4d89560b307d)